src/test/java/com/moabam/api/presentation/MemberControllerTest.java

```diff
@@ -417,4 +428,80 @@ void search_member_failBy_default_skin_size() throws Exception {
 			.andExpect(status().is4xxClientError());
 
 	}
+
+	@DisplayName("회원 정보 요청 성공")
+	@WithMember
+	@ParameterizedTest
+	@CsvSource({"intro, null", "null, nickname", "null, null", "intro, nickname"})
```

Comment:
```suggestion
	@CsvSource({"intro,", ", nickname", ",", "intro, nickname"})
```
C: 값을 안 넣어주면 null로 들어가는 걸로 알고 있습니다!

Reply:
헉 처음 알았습니다